ucm/sof-bytcht-da7213/HiFi

```diff
@@ -0,0 +1,185 @@
+		cset "name='Mic 1 Volume' 6"
+		cset "name='Mixin PGA Volume' 47%"
+		cset "name='ADC Volume' 112"
+
+		cset "name='Mixin Left Mic 1 Switch' on"
+		cset "name='Mixin Right Mic 1 Switch' on"
+		cset "name='Mixin PGA Switch' on"
+		cset "name='Mic 1 Switch' on"
+		cset "name='ADC Switch' on"
+		cset "name='Headset Mic Switch' on"
+	]
+
+	DisableSequence [
+		cdev "hw:sofbytchtda7213"
+
+		cset "name='Headset Mic Switch' off"
+		cset "name='ADC Switch' off"
+		cset "name='Mic 1 Switch' off"
+		cset "name='Mixin PGA Switch' off"
+		cset "name='Mixin Left Mic 1 Switch' off"
+		cset "name='Mixin Right Mic 1 Switch' off"
+	]
+
+	Value {
+		CaptureChannels 2
+	}
+}
+
+SectionDevice."MIC2" {
+	Comment "MIC2 capture"
+
+	ConflictingDevice [
+		"MIC1"
+		"AUX"
+	]
+
+	EnableSequence [
+		cdev "hw:sofbytchtda7213"
+
+#		cset "name='Mic 2 Amp Source MUX' Differential"
+#		cset "name='Mic 2 Amp Source MUX' MIC_N"
+
+		cset "name='Mic 2 Amp Source MUX' MIC_P"
+		cset "name='Mic 2 Volume' 6"
+		cset "name='Mixin PGA Volume' 47%"
+		cset "name='ADC Volume' 112"
+
+		cset "name='Mixin Left Mic 2 Switch' on"
+		cset "name='Mixin Right Mic 2 Switch' on"
+		cset "name='Mixin PGA Switch' on"
+		cset "name='Mic 2 Switch' on"
+		cset "name='ADC Switch' on"
+		cset "name='Mic Switch' on"
+
+	]
+
+	DisableSequence [
+		cdev "hw:sofbytchtda7213"
+
+		cset "name='Mic Switch' off"
+		cset "name='ADC Switch' off"
+		cset "name='Mic 1 Switch' off"
+		cset "name='Mixin PGA Switch' off"
+		cset "name='Mixin Left Mic 1 Switch' off"
+		cset "name='Mixin Right Mic 1 Switch' off"
+	]
+
+	Value {
+		CaptureChannels 2
+	}
+}
+
+SectionDevice."AUX" {
+	Comment "AUX capture"
+
+	ConflictingDevice [
+		"MIC1"
+		"MIC2"
+	]
+
+	EnableSequence [
+		cdev "hw:sofbytchtda7213"
+
+		cset "name='Aux Volume' 78%"
+		cset "name='ADC Volume' 76%"
+		cset "name='Mixin PGA Volume' 47%"
+
+		cset "name='Mixin Left Aux Left Switch' on"
+		cset "name='Mixin Right Aux Right Switch' on"
+		cset "name='Mixin PGA Switch' on"
+		cset "name='ADC Switch' on"
+		cset "name='Aux Switch' on"
+	]
+
+	DisableSequence [
+		cdev "hw:sofbytchtda7213"
+
+		cset "name='Aux Switch' off"
+		cset "name='ADC Switch' off"
+		cset "name='Mixin PGA Switch' off"
+		cset "name='Mixin Left Aux Left Switch' off"
+		cset "name='Mixin Right Aux Right Switch' off"
+	]
+
+	Value {
+		CaptureChannels 2
+	}
+}
+
+SectionDevice."Headphones" {
+	Comment "Headphones playback"
+
+	EnableSequence [
+		cdev "hw:sofbytchtda7213"
+
+		cset "name='Headphone Volume' 50%"
+		cset "name='Mixout Left DAC Left Switch' on"
+		cset "name='Mixout Right DAC Right Switch' on"
+		cset "name='Headphone Switch' on"
+		cset "name='Headphone Jack Switch' on"
+
+	]
+
+	DisableSequence [
+		cdev "hw:sofbytchtda7213"
+
+		cset "name='Headphone Jack Switch' off"
+		cset "name='Headphone Switch' off"
+		cset "name='Mixout Left DAC Left Switch' off"
+		cset "name='Mixout Right DAC Right Switch' off"
+	]
+
+	Value {
+		PlaybackChannels "2"
```

Comment:
we need add  something like:
```c
PlaybackPCM "hw:sofbytchtda7213, num" 
JackControl "Headphone xxx" \\ for pulse audio to detect headphone
```

Reply:
@RanderWang can you check if there is a working platform with this codec in Shanghai?

I don't know if the machine driver reports the jack so it may not be really important. this is mostly a test rig.

Reply:
@plbossart  I checked but didn’t find one. So we don't need to set JackControl now since we can't test it.